logger.gd

```diff
@@ -441,11 +443,33 @@ func get_default_output_level():
 # Output formatting
 # -----------------
 
-static func format(template, level, module, message):
```

Comment:
Why remove the `static`?

Reply:
I see this is needed to access the member var. Not sure why this would need to be static anyway, so I guess we can remove both.